scripts/contract.mjs

```diff
@@ -1,19 +1,19 @@
 import util from "util";
 import { exec } from "child_process";
 import dotenv from "dotenv";
-import {Interface} from '@ethersproject/abi';
+import { Interface } from "@ethersproject/abi";
```

Comment:
I wonder if we should disable automatic reformat here, because if we ever wanted to create a PR upstream it would cause issues.

Reply:
I like your thinking, I'll look into that tomorrow. 🔍

Reply:
how does auto reformatting work, btw?  my system isn't auto-formatting from what i can tell.